spec/spec.md

```diff
@@ -251,72 +208,7 @@ requirement.
 
 ::: example Presentation Definition - Single Group Example
 ```json
-{
-  // VP, OIDC, DIDComm, or CHAPI outer wrapper
-  "presentation_definition": {
-    "id": "32f54163-7166-48f1-93d8-ff217bdb0653",
-    "submission_requirements": [{
-      "name": "Citizenship Information",
-      "rule": "pick",
-      "count": 1,
-      "from": "A"
-    }],
-    "input_descriptors": [
-      {
-        "id": "citizenship_input_1",
-        "name": "EU Driver's License",
-        "group": ["A"],
-        "schema": [
-          {
-            "uri": "https://eu.com/claims/DriversLicense.json"
-          }
-        ],
-        "constraints": {
-          "fields": [
-            {
-              "path": ["$.issuer", "$.vc.issuer", "$.iss"],
-              "purpose": "The claim must be from one of the specified issuers",
-              "filter": {
-                "type": "string",
-                "pattern": "did:example:gov1|did:example:gov2"
-              }
-            },
-            {
-              "path": ["$.credentialSubject.dob", "$.vc.credentialSubject.dob", "$.dob"],
-              "filter": {
-                "type": "string",
-                "format": "date",
-                "maximum": "1999-6-15"
-              }
-            }
-          ]
-        }
-      },
-      {
-        "id": "citizenship_input_2",
-        "name": "US Passport",
-        "group": ["A"],
-        "schema": [
-          {
-            "uri": "hub://did:foo:123/Collections/schema.us.gov/passport.json"
-          }
-        ],
-        "constraints": {
-          "fields": [
-            {
-              "path": ["$.credentialSubject.birth_date", "$.vc.credentialSubject.birth_date", "$.birth_date"],
-              "filter": {
-                "type": "string",
-                "format": "date",
-                "maximum": "1999-5-16"
-              }
-            }
-          ]
-        }
-      }
-    ]
-  }
-}
+[[import: ../test/presentation-definition/single_group_example.json]]
```

Comment:
Same comment above, the form of this is going to be: [[insert: ./test/presentation-definition/single_group_example.json]]